### PR TITLE
welcome

### PR DESCRIPTION
**Pull request by zbingoz**

### CODE REVIEW

**Review by haibinzang — APPROVED** [review]
good

**Review by zbingoz — COMMENTED** [review]
very good

File: welcome.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+welcome
```
Comment (zbingoz) [inline welcome.txt:1]: that is hello ?

**Review by haibinzang — APPROVED** [review]
fine

**Review by haibinzang — APPROVED** [review]
ok

**Review by zbingoz — COMMENTED** [review]
abc

**Review by BINGOGO123 — APPROVED** [review]
abc

**Pull request merged**